staff/jaume-serradell/spotify-app/scripts/main.js

```diff
@@ -0,0 +1,94 @@
+// my presentation logic
```

Comment:
You should use 'use strict' at the begining

Reply:
'use stricti' mf

Reply:
Done!